File: main/src/main/java/net/fwitz/math/main/binary/split/functions/trig/SplitCosZ.java
package net.fwitz.math.main.binary.split.functions.trig;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitCosZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("cos(z)")
                .computeFn(SplitComplex::cos)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/trig/SplitCosZ.kt
package net.fwitz.math.main.binary.split.functions.trig

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitCosZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("cos(z)")
        .computeFn { z -> z.cos }
        .render()
}